pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -95,13 +95,21 @@ type BundleTarget struct {
 	// ConfigMap is the target ConfigMap in Namespaces that all Bundle source
 	// data will be synced to.
 	ConfigMap *KeySelector `json:"configMap,omitempty"`
+	// AdditionalFormats specifies any additional formats to write to the configmap
+	// +optional
```

Comment:
nitpick: We might add support for other targets, so it'd be good to avoid talking about ConfigMaps here!

```suggestion
	// AdditionalFormats specifies any additional formats to write to the target
	// +optional
```